nuggetsapp/urls.py

```diff
@@ -1,7 +1,10 @@
-from django.conf.urls import url
+from django.conf.urls import url, include
 from . import views
 
 urlpatterns = [
+    url(r'^api/v1/nuggets/(?P<user_id>[0-9]+)/$', views.nuggets_for_user, name='nuggets_for_user'),
```

Comment:
The way I imagined this is that we'd have endpoints for (at the minimum):
1. nuggets_for_user
2. single nugget
3. user info (maybe)

So I'm not super familiar with REST API best practices in terms of route structure, but my best guess for the above would be:
1. `api/v0/user/123/nuggets` (<---- this one I'm really not sure what best practice would be.  Maybe that is weird and instead we should do `api/v0/nugget_for_user/123`? But this seems anti-REST.)
2. `api/v0/nugget/789`
3. `api/v0/user/123`

I think we should update what you have (`api/v0/nuggets/<user_id>`), since I think a tenet of REST is that it is obvious what is being passed in, but in this case it's not clear that `/nuggets` is expecting a user_id.  But let me know what you find on the internets about REST best practices, would be very curious to know.